src/main/java/org/jenkinsci/plugins/casc/integrations/projectmatriaxauth/ProjectMatrixAuthorizationStrategyConfigurator.java

```diff
@@ -0,0 +1,55 @@
+package org.jenkinsci.plugins.casc.integrations.projectmatriaxauth;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import hudson.Extension;
+import hudson.security.Permission;
+import hudson.security.ProjectMatrixAuthorizationStrategy;
+import org.jenkinsci.plugins.casc.Attribute;
+import org.jenkinsci.plugins.casc.Configurator;
+import org.jenkinsci.plugins.casc.RootElementConfigurator;
+import org.jenkinsci.plugins.casc.integrations.globalmatrixauth.GroupPermissionDefinition;
+
+import java.util.*;
+
+@Extension(optional = true)
```

Comment:
If this plugin is intended to become an "API" kind of plugin, perhaps `@Restricted`

Reply:
:+1: